importqueue/importqueue_test.go

```diff
@@ -263,6 +301,30 @@ func TestQueueCantabularFile(t *testing.T) {
 				CantabularType: formatCantabularTable,
 			})
 		})
+
+		Convey("Then importing a 'cantabular_flexible_table' recipe sends the expected import event to the cantabular queue", func() {
```

Comment:
Should this say something like:
Convey("Then importing a 'cantabular_flexible_table' recipe with a 'correct' instanceID sends the expected import event to the cantabular queue", func() {